service/history/workflow_task_handler.go

```diff
@@ -351,7 +355,8 @@ func (handler *workflowTaskHandlerImpl) handleMessage(
 				enumspb.WORKFLOW_TASK_FAILED_CAUSE_BAD_UPDATE_WORKFLOW_EXECUTION_MESSAGE,
 				serviceerror.NewNotFound(fmt.Sprintf("update %q not found", message.ProtocolInstanceId)))
 		}
-		if err := upd.OnMessage(ctx, message, workflow.WithEffects(handler.effects, handler.mutableState)); err != nil {
+
+		if err := upd.OnMessage(ctx, message, handler.mutableState.IsWorkflowExecutionRunning(), workflow.WithEffects(handler.effects, handler.mutableState)); err != nil {
```

Comment:
Do we have limit on number of characters in line? Anyway, people read faster at around 80 and common agreement in code is around 120. I've heard that it is impolite to write lines which make people to scroll right to read.

Reply:
We don't have any rules on this and I personally try not to cut lines if it is not necessary. I am happy to adopt any rule. Do you think that 120 is some sort of standart?